yarn.lock

```diff
@@ -277,7 +277,7 @@ __metadata:
     crypto-browserify: 3.12.0
     date-fns: 4.1.0
     deprecated-react-native-prop-types: 5.0.0
-    detox: 20.27.3
+    detox: 20.27.2
```

Comment:
did you mean to downgrade detox version?

Reply:
Yes the official latest one is 20.27.2 - https://github.com/wix/Detox/releases/tag/20.27.2